src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ImportController.php

```diff
@@ -258,6 +378,27 @@ public function downloadSampleAction($sampleName)
         return $response;
     }
 
+    /**
+     * Process the import.
+     *
+     * @DemoRestricted(redirectRoute="admin_import")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function processImportAction(Request $request)
```

Comment:
Is this temporary ?

Reply:
actually this action is not even used anymore, since the import is processed via an ajax call to legacy controller. I'll remove it :)